vscode-extension/package.json

```diff
@@ -128,6 +128,11 @@
           "type": "string",
           "default": "",
           "description": "Version of the AIConfig Editor extension last time it was activated. We use this value to check if the extension has been updated to prompt users to reload VS Code"
+        },
+        "vscode-aiconfig.env_file_path": {
+          "type": "string",
+          "default": "",
+          "description": "env_file_path"
```

Comment:
Description should be clearer on what this is for

Reply:
Updated to "File path to your .env file containing your API keys"